dashboard/app/views/levels/editors/_poetry.html.haml

```diff
@@ -1,15 +1,22 @@
+:ruby
+  warnings = {
+    level_type: "Example solutions are dependent on the level type field for generating solution URLs. If you change the level type, any existing example solution URLs will break.",
+    encrypted_examples: "For Poetry levels, the project URL for example solutions depends on the level type. Any existing example solutions will break if you change the level type without updating the channel tokens here."
```

Comment:
Could we clarify this warning when we say "depends on level type"? I want it to be clear that we're just talking about "Poetry" vs "Poetry HOC" and not about somehow changing from Poetry to a level type with a different lab.

Ugh, I think the issue is that we suddenly have two meanings for "level type". I think most writers interpret "level type" as things like Poetry/Spritelab/Applab/Video/etc, but in Poetry levels, it _also_ means one of the two modes, "Poetry" (module) and "Poetry HOC". I think I'd go so far as to say that "level type" is the wrong description for changing between these two modes. I'd like for us to look at renaming this setting later, but also clarify the warning for writers now.

Reply:
yeah, agreed that that's really unclear. in the code we refer to this field as `standalone_app_name` -- it's only labeled as "Level Type" in this UI. would some formulation of the actual field be clearer? something like "poetry standalone app type" or "poetry app type"?

this won't affect any code so i think we can go with whatever is clearest for the UI here

Reply:
poetry level subtype?

Reply:
"Poetry App Type" or "Poetry Level Subtype" work for me. Could we update the warning to include the two possible options like this?
> For Poetry levels, the project URL for example solutions depends on the level subtype (Poetry or Poetry HOC).

I think this clarification would help for levelbuilders who are unfamiliar with this new term.

Reply:
updated in [this commit](https://github.com/code-dot-org/code-dot-org/pull/43667/commits/a3be724cec71930b924a172b8fc6e0269910f9a3)! (screenshot in PR description is updated too so you can see how it all looks together)

Reply:
thanks - lgtm!